src/pages/workspace/accounting/netsuite/export/NetSuitePreferredExporterSelectPage.tsx

```diff
@@ -0,0 +1,104 @@
+import {isEmpty} from 'lodash';
+import React, {useCallback, useMemo} from 'react';
+import {View} from 'react-native';
+import RadioListItem from '@components/SelectionList/RadioListItem';
+import type {ListItem} from '@components/SelectionList/types';
+import SelectionScreen from '@components/SelectionScreen';
+import Text from '@components/Text';
+import useCurrentUserPersonalDetails from '@hooks/useCurrentUserPersonalDetails';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import * as Connections from '@libs/actions/connections/NetSuiteCommands';
+import {getAdminEmployees, isExpensifyTeam} from '@libs/PolicyUtils';
+import Navigation from '@navigation/Navigation';
+import type {WithPolicyConnectionsProps} from '@pages/workspace/withPolicyConnections';
+import withPolicyConnections from '@pages/workspace/withPolicyConnections';
+import CONST from '@src/CONST';
+import ROUTES from '@src/ROUTES';
+
+type CardListItem = ListItem & {
+    value: string;
+};
+
+function NetSuitePreferredExporterSelectPage({policy}: WithPolicyConnectionsProps) {
+    const config = policy?.connections?.netsuite.options.config;
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+    const policyOwner = policy?.owner ?? '';
+    const exporters = getAdminEmployees(policy);
+    const {login: currentUserLogin} = useCurrentUserPersonalDetails();
+
+    const policyID = policy?.id ?? '-1';
+    const data: CardListItem[] = useMemo(() => {
+        if (!isEmpty(policyOwner) && isEmpty(exporters)) {
+            return [
+                {
+                    value: policyOwner,
+                    text: policyOwner,
+                    keyForList: policyOwner,
+                    isSelected: true,
+                },
+            ];
+        }
+
+        return exporters?.reduce<CardListItem[]>((options, exporter) => {
+            if (!exporter.email) {
+                return options;
+            }
+
+            // Don't show guides if the current user is not a guide themselves or an Expensify employee
```

Comment:
As a cleanup we could merge this with other codes. I think I've seen this in QBO/Xero.